data/json/effects_on_condition/npc_eocs/hallucination_eocs.json

```diff
@@ -0,0 +1,195 @@
+      {
+        "weighted_list_eocs": [
+          [ "minor_hallucinations", { "const": 10 } ],
+          [ "intermediate_hallucinations", { "const": 6 } ],
+          [ "bad_hallucinations", { "const": 4 } ]
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "minor_hallucinations",
+    "effect": [
+      {
+        "weighted_list_eocs": [
+          [ "paranoid", { "const": 75 } ],
+          [ "grandiose", { "const": 25 } ],
+          [ "self_talk", { "const": 14 } ],
+          [ "weapon_talk", { "const": 12 } ],
+          [ "weapon_talk_you_bleeding", { "const": 12 } ],
+          [ "weapon_talk_monsters_nearby", { "const": 12 } ],
+          [ "false_npc_talk", { "const": 10 } ]
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "intermediate_hallucinations",
+    "effect": [
+      {
+        "weighted_list_eocs": [
+          [ "self_shout", { "const": 30 } ],
+          [ "hear_false_sound", { "const": 20 } ],
+          [ "formication", { "const": 20 } ],
+          [ "numbness", { "const": 15 } ],
+          [ "shaking", { "const": 10 } ]
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "bad_hallucinations",
+    "effect": [ { "weighted_list_eocs": [ [ "creature_hallucinations", { "const": 20 } ], [ "visuals", { "const": 10 } ] ] } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "paranoid",
+    "effect": [
+      { "u_message": "schizo_delusion_paranoid", "snippet": true },
+      {
+        "u_add_morale": "morale_feeling_bad",
+        "bonus": -20,
+        "max_bonus": -100,
+        "duration": "30 minutes",
+        "decay_start": "30 minutes"
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "self_talk",
+    "effect": { "u_make_sound": "schizo_self_talk", "volume": 5, "snippet": true, "type": "speech" }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "grandiose",
+    "effect": [
+      { "u_message": "schizo_delusion_grandiose", "snippet": true },
+      {
+        "u_add_morale": "morale_feeling_good",
+        "bonus": 20,
+        "max_bonus": 100,
+        "duration": "30 minutes",
+        "decay_start": "30 minutes"
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "weapon_talk",
+    "effect": {
+      "u_run_inv_eocs": "random",
+      "search_data": [ { "wielded_only": true } ],
+      "true_eocs": [
+        {
+          "id": "injured_talk",
+          "condition": { "math": [ "n_hp('ALL')", "<", "n_hp_max('bp_null')" ] },
+          "effect": { "u_message": "Your weapon says \"<schizo_weapon_talk_damaged>\"" },
+          "false_effect": { "run_eocs": { "id": "normal_talk", "effect": { "u_message": "Your weapon says \"<schizo_weapon_talk_misc>\"" } } }
+        }
+      ]
+    },
+    "false_effect": { "run_eocs": "minor_hallucinations" }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "weapon_talk_you_bleeding",
```

Comment:
I cannot get this to fire even when I activate the eoc while holding a sword and bleeding profusely.  I bled myself to the point that teh character is probably dying from blood loss.